lib/transaction.py

```diff
@@ -326,6 +326,7 @@ def parse_input(vds):
             d['address'] = UnknownAddress()
             d['type'] = 'unknown'
         if not Transaction.is_txin_complete(d):
+            del d['scriptSig']
```

Comment:
Considered doing `d['scriptSig'] = None`, however a missing key more closely resembles how unsigned transactions look like when made using `wallet.make_unsigned_transaction` for instance.

Reply:
Right that's probably the right call here. I will also independently check that nothing funny can happen as a result of this improvement .. but yes that's right it would more closely resemble unsigned tx's to do that....